code/API_definitions/S2CVPN4(1).yaml

```diff
@@ -0,0 +1,936 @@
+    * Duration Duration (in hours) for which the Site to Cloud VPN session (between CE and cloud PE) should be created. This parameter is optional. When not specified, a default session duration (e.g. 6 months) is applied. The user may request a termination before its expiration.
+    * Physical Location Physical location of the Customer Edge Router.
+
+
+    NOTE- The private physical link between CE and cloud PE is default deployed for one click calling, otherwise it is required to install the physical link, which is out of the scope of this API.
+
+  version: '0.1'
+servers:
+- url: https://localhost:8080/webapi
+  description: develop server
+paths: 
+    /S2C_VPN_Assessment:
+     post:
+       parameters:
+        - $ref: '#/components/parameters/X-Correlator'
+       requestBody:
+        content:
+          application/json:
+           schema:
+            type: object
+            properties:
+             isProtected:
+               type: bool
+               description: Is Site to Cloud VPN protected
+             CEType:
+              type: string
+              description: Site to Cloud VPN access point type
+              enum: 
+                 - site
+                 - cloud
+             primaryCEIP:
+              description: primary Access point CE address
+              type: string
+              format: ipv4
+              example: "84.125.93.10"
+             secondaryCEIP:
+              description: secondary Access point CE address
+              type: string
+              format: ipv4
+              example: "84.125.94.10"
+             PEType:
+              type: string
+              description: Site to Cloud VPN endpoint type ( Cloud PE/gateway type)
+              enum: 
+                 - site
+                 - cloud
+             primaryPEIP:
+              description: primary Endpoint Cloud PE address
+              type: string
+              format: ipv4
+              example: "84.125.93.10"
+             secondaryPEIP:
+              description:  secondary Endpoint Cloud PE address
+              type: string
+              format: ipv4
+              example: "84.125.94.10"
+             guaranteeBandwidth:
+              type: integer
+              description: Guarantee Bandwidth (Mps)
+              minimum: 1
+             duration:
+              type: integer
+              description: "Guarantee duration, unit:Hour"
+             CELocation:
+              type: string
+       responses:
+        '200':
+         description: Successful response
+         headers:
+          X-Correlator:
+           description: When the API Consumer includes the "X-Correlator" header in the request, the API provider must include it in the response with the same value that was used in the request
+           schema:
+            type: integer

```

Comment:
Please cf https://github.com/camaraproject/Commonalities/blob/main/documentation/API-design-guidelines.md#9-architecture-headers:
- x-correlator instead of X-Correlator
- format is string, not integer
- there is a schema defined within Components section